docs/DevelopmentSetupAndroid.md

```diff
@@ -78,3 +78,29 @@ Genymotion is much easier to set up than stock Google emulators. However, it's o
   ![Create AVD dialog](/react-native/img/CreateAVD.png)
   2. With the new AVD selected, click `Start...`
 5. To bring up the developer menu press F2
+
+### Windows Hyper-V Alternative
+
+A lot of developers, especially in the enterprise space are using Hyper-V on their machines. Unfortunately the default Android Emulator cannot be hardware accelerated if Hyper-V is turned on and turning it off is often not an option.
+
+Fortunately there is a great and free android emulator that uses Hyper-V.
```

Comment:
The [Visual Studio Emulator for Android](https://www.visualstudio.com/en-us/features/msft-android-emulator-vs.aspx) is a free android emulator that is hardware accelerated via Hyper-V. It doesn't require you to install Visual Studio at all.